docs/dev/table/connectors/print.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Print SQL Connector"
+nav-title: Print
+nav-parent_id: sql-connectors
+nav-pos: 5
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<span class="label label-primary">Sink: Bounded</span>
+<span class="label label-primary">Sink: UnBounded</span>
+
+* This will be replaced by the TOC
+{:toc}
+
+The Print connector allows for writing every row to the standard output or standard error stream.
+
+It is designed for:
+
+- Easy test for streaming job.
+- Very useful in production debugging.
+
+Four possible format options:
+
+- PRINT_IDENTIFIER:taskId> output  <- PRINT_IDENTIFIER provided, parallelism > 1
+- PRINT_IDENTIFIER> output         <- PRINT_IDENTIFIER provided, parallelism == 1
+- taskId> output         		   <- no PRINT_IDENTIFIER provided, parallelism > 1
+- output                 		   <- no PRINT_IDENTIFIER provided, parallelism == 1
+
+The output string format is "$RowKind(f0,f1,f2...)", example is: "+I(1,1)".
+
+The Print connector is built-in.
+
+<span class="label label-danger">Attention</span> Print sink print records in tasks, you need to observe the task log.
+
+How to create an Print table
+----------------
+
+Although it doesn't make sense to define the fields of print table, you need to write them all in DDL.
+
+<div class="codetabs" markdown="1">
+<div data-lang="SQL" markdown="1">
+{% highlight sql %}
+CREATE TABLE print_table (
+ f0 INT,
+ f1 INT,
+ f2 STRING,
+ f3 DOUBLE
+) WITH (
+ 'connector' = 'print'
+)
+{% endhighlight %}
+</div>
+</div>
+
+Another way is using [LIKE Clause]({{ site.baseurl }}/dev/table/sql/create.html#create-table).
```

Comment:
```suggestion
Alternatively, it may be based on an existing schema using the [LIKE Clause]({{ site.baseurl }}/dev/table/sql/create.html#create-table).
```